app/src/main/res/layout/main_activity.xml

```diff
@@ -23,10 +23,10 @@
         <EditText
             android:id="@+id/textView"
             style="@style/TextNumber.Input"
-            android:layout_width="0dp"
+            android:layout_width="@dimen/dimen_zero"
             android:layout_height="wrap_content"
             android:layout_marginTop="30dp"
```

Comment:
No need to use hardcoded values in your xml. Try to move this value to resources.